test/client/js/libs/handlebars/tojson.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+const expect = require("chai").expect;
+const tojson = require("../../../../../client/js/libs/handlebars/tojson");
+
+describe("tojson Handlebars helper", function() {
+	it("should return any number under 1000", function() {
```

Comment:
copy paste error on the test description ;)

Reply:
Good catch! Fixed, thanks!